livebooks/create-a-swiftui-application.livemd

```diff
@@ -113,21 +111,21 @@ Open Xcode and select Create New Project.
 
 <!-- livebook:{"break_markdown":true} -->
 
-![Xcode Create New Project](https://github.com/liveview-native/documentation_assets/blob/main/xcode-create-new-project.png?raw=true)
+![Xcode Create New Project](https://i.imgur.com/BRN1G7F.png)
 
 <!-- livebook:{"break_markdown":true} -->
 
 To create an iOS application, select the **iOS** and **App** options and click **Next**.
 
 <!-- livebook:{"break_markdown":true} -->
 
-![Xcode Create Template For New Project](https://github.com/liveview-native/documentation_assets/blob/main/xcode-create-template-for-new-project.png?raw=true)
+![Xcode Create Template For New Project](https://i.imgur.com/hrk69Ew.png)
```

Comment:
we shouldn't be hosting on imgur. I think that's going to bite us

Reply:
that's not correct, we should be hosting all image assets on blog posts on the dockyard asset server uploaded in the Beacon admin.

Reply:
I looked into other livebooks and the Nerves project is hosting from github: https://github.com/nerves-livebook/nerves_livebook/blob/main/priv/samples/basics/blink.livemd

Reply:
I was incorrect about our blog process, you're right. I'll move the assets back to using github for hosting and also move any images using IMGUR to github.

I think we should use the separate documentation_assets project since otherwise the images won't be available until the PR is merged.

Reply:
@bcardarella Done!